database/schema.sql

```diff
@@ -3,9 +3,31 @@ CREATE DATABASE Sip;
 
 USE Sip;
 
-CREATE TABLE profiles (
-    Id INT NULL AUTO_INCREMENT PRIMARY KEY,
-    user_name VARCHAR(255) NOT NULL,
-    location VARCHAR(255) NOT NULL,
-    drink VARCHAR(255) NOT NULL
-);
\ No newline at end of file
+CREATE TABLE Profiles (
+    Id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    Username VARCHAR(255) NOT NULL,
+    Location VARCHAR(255) NOT NULL,
+    Userpassword VARCHAR(255) NOT NULL,
+    Profileimages VARCHAR (255) NOT NULL
+);
+
+CREATE TABLE Reviews(
+    Id INT NOT NULL PRIMARY KEY,
+    Namedrink VARCHAR(255) NOT NULL,
+    Store VARCHAR(255) NOT NULL,
+    Rating INT (255) NOT NULL,
+    Comments VARCHAR(255) NOT NULL,
```

Comment:
review Line 9, 
change the names from location to something else, 
the color red means that it is a reserved keyword.
Make the names "camelCase".